src/modules/missionHelper/i18n/de_DE.root.json

```diff
@@ -81,10 +81,6 @@
             "description": "Zeigt an, wenn der DHuFüKW nur gefordert wird, wenn vorhanden.",
             "title": "Optionale Diensthunde"
         },
-        "max_civil_patrol_replace_police_cars": {
```

Comment:
They are sorted alphabetical in the building process

Reply:
please explain this bug

Reply:
aaaand you're right again :D